core/src/scss/utilities/mixins/link/_action-link.scss

```diff
@@ -1,33 +0,0 @@
-@charset "UTF-8";
-
-//
-// @action-link($color: $color-bright-blue)
-//
-// Provides a right carat link with alternate color.
-//
-// $color - Default color. Defaults to $color-bright-blue.
-// $hover - Hover color. Defaults to $color-black.
-//
-// Style guide: Mixins.Link.action-link
-//
-@mixin action-link($color: color(link), $hover: color(link--hover)) {
```

Comment:
Instead of completely removing the function, please deprecate it and patch through to the new option.

Reply:
Done